.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on: [push]
+jobs:
+  setup-python:
```

Comment:
The job shouldn't be called "setup-python" but maybe "CI" or "check" or something like that.